src/components/event.js

```diff
@@ -0,0 +1,52 @@
+//Individual Event contained within our Events page
```

Comment:
Can this page live as a sibling to the events index.js page? They can be in a folder in the `/pages` directory.